asreviewcontrib/insights/plot.py

```diff
@@ -70,7 +70,7 @@ def _plot_recall(ax, labels, x_absolute=False, y_absolute=False):
 
     ax.step(x, y, where='post')
     ax.set_title("Recall")
-    ax.set(xlabel='#', ylabel='Recall')
+    ax.set(xlabel='Proportion total number of records', ylabel='Recall')
```

Comment:
This label should be dependent on argument `x_absolute`. If you come up with a description for the absolute axis, then I will implement the switch.

Reply:
I expected such a question :-) 
What about "Number of records"?